var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -195,6 +197,10 @@ class Petsc(Package, CudaPackage, ROCmPackage):
     # older versions of petsc did not support mumps when +int64
     conflicts('+mumps', when='@:3.12+int64')
 
+    # conflicts with ~fortran, these packages require +fortran
+    conflicts('+scalapack', when='~fortran')
+    conflicts('+superlu-dist', when='~fortran')
```

Comment:
Perhaps superlu_dist can also provide this variant.

Reply:
> Perhaps superlu_dist can also provide this variant.

I mean to cc: @liuyangzhuan

Reply:
Agree, but it might be easier to address that in a separate PR.

Reply:
Yeah - that could be a separate PR